content/docs/csidriver/release/powerflex.md

```diff
@@ -6,19 +6,17 @@ description: Release notes for PowerFlex CSI driver
 ## Release Notes - CSI PowerFlex v2.8.0
 
 
+
 ### New Features/Changes
 
 - [#724 - [FEATURE]: CSM support for Openshift 4.13](https://github.com/dell/csm/issues/724)
 - [#763 - [FEATURE]: CSI-PowerFlex 4.0 NFS support](https://github.com/dell/csm/issues/763)
 - [#876 - [FEATURE]: CSI 1.5 spec support -StorageCapacityTracking](https://github.com/dell/csm/issues/876)
-- [#926 - [FEATURE]: Set up golangci-lint for all CSM repositories](https://github.com/dell/csm/issues/926)
 - [#947 - [FEATURE]: K8S 1.28 support in CSM 1.8](https://github.com/dell/csm/issues/947)
-- [#878 - [FEATURE]: CSI 1.5 spec support : Implement Volume Limits](https://github.com/dell/csm/issues/878)
-- [#885 - [FEATURE]: SDC 3.6.1 support](https://github.com/dell/csm/issues/885)
```

Comment:
This also should not get removed.

Reply:
@VamsiSiddu-7 Similarly, the milestone hasn't been added here either. I kindly request that each team takes the initiative to add it from their end. I'm unsure if this is considered part of the CSM 1.8 deliverable, so your assistance in this case is greatly appreciated.